src/app/clusters/closure-control-server/closure-control-cluster-logic.cpp

```diff
@@ -504,15 +504,32 @@ Protocols::InteractionModel::Status ClusterLogic::HandleMoveTo(Optional<TargetPo
 
     VerifyOrReturnError(position.HasValue() || latch.HasValue() || speed.HasValue(), Status::InvalidCommand);
 
+    DataModel::Nullable<GenericOverallCurrentState> overallCurrentState;
     DataModel::Nullable<GenericOverallTargetState> overallTargetState;
     VerifyOrReturnError(GetOverallTargetState(overallTargetState) == CHIP_NO_ERROR, Status::Failure);
+    VerifyOrReturnError(GetOverallCurrentState(overallCurrentState) == CHIP_NO_ERROR, Status::Failure);
+    VerifyOrReturnError(!overallCurrentState.IsNull(), Status::InvalidInState,
+                        ChipLogError(AppServer, "OverallCurrentState is null over endpoint : %d", mMatterContext.GetEndpointId()));
```

Comment:
```suggestion
                        ChipLogError(AppServer, "OverallCurrentState is null for endpoint %d", mMatterContext.GetEndpointId()));
```
is probably better.  Or "on endpoint"?

Reply:
modified it to on endpoint